config.py

```diff
@@ -20,3 +25,37 @@
 DOT_TWO_ROLE = 1078208892296761404
 DOT_THREE_ROLE = 1078208973326536724
 SPAM_ROLE = 1078208518793994240
+
+LIVE_SESSION_PING_ROLE = None if ENVIRONMENT == "prod" else 1131210834744447067
```

Comment:
Rule of 3 says this environment check should be probably be its own helper, or just load different variables at startup instead of checking each time.

Reply:
Should I add an environ.get at the beginning of the config then?